dd-trace-api/src/main/java/datadog/trace/api/config/TracerConfig.java

```diff
@@ -73,7 +73,8 @@ public final class TracerConfig {
   public static final String TRACE_HTTP_CLIENT_ERROR_STATUSES = "trace.http.client.error.statuses";
 
   public static final String SPLIT_BY_TAGS = "trace.split-by-tags";
-
+  // trace latency interceptor value should be in ms
+  public static final String TRACE_KEEP_LATENCY_THRESHOLD_MS = "trace.keep.latency.threshold.ms";
```

Comment:
[nit] Can we have this marked as "experimental" since this can lead to broken/incomplete traces?

```suggestion
  public static final String TRACE_KEEP_LATENCY_THRESHOLD_MS = "trace.experimental.keep.latency.threshold.ms";
```